spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume_spec.rb

```diff
@@ -0,0 +1,37 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::StorageManager::CloudVolume do
+  let(:ems) do
+    FactoryBot.create(:ems_ibm_cloud_power_virtual_servers_cloud, :provider_region => "us-south")
+  end
+  let(:StorageManager) { FactoryBot.create(:ems_IbmCloud_StorageManager, :parent_ems_id => ems_cloud.id) }
+  let(:cloud_volume) { FactoryBot.create(:cloud_volume_IbmCloud, :ext_management_system => StorageManager ) }
```

Comment:
Also need to define a cloud_volume factory (similar to `:vm_ibm_cloud_power_vritual_servers`, https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/spec/factories/vm.rb).